dashboard/app/models/levels/dancelab.rb

```diff
@@ -52,9 +52,9 @@ def self.create_from_level_builder(params, level_params)
   def common_blocks(type)
   end
 
-  # Manually curated
-  # TODO - epeach - manually populate these values from song manifest
   def self.hoc_songs
-    [["MC Hammer - U Can't Touch This", "hammer"], ["Macklemore - Can't Hold Us", "macklemore90"], ["The Black Eyed Peas - I Got a Feeling", "peas"]]
+    manifest_json = AWS::S3.create_client.get_object(bucket: 'cdo-sound-library', key: 'hoc_song_meta/songManifest.json')[:body].read
+    manifest = JSON.parse(manifest_json)
+    manifest['songs'].map {|song| [song['text'], song['id']]}
```

Comment:
Non-blocking, but I think we should add the pg-13 attribute to this display so levelbuilders don't make a pg-13 song default.